src/sql/orderbook/latest_block.sql

```diff
@@ -1,3 +1,2 @@
-select min(block_number) latest
-from settlements
-where tx_from is null;
\ No newline at end of file
+select max(block_number) latest
```

Comment:
Should we maybe have
`max(block_number) + 1`
?

Not sure if it makes any sense, but so as to follow the convention used up till now that the latest block is to be ignored (i.e., it is an open inverval)

Reply:
I am not sure either what makes sense here. From what I can piece together is that the original query returned a block which did not have data in our database and subtracted `65`. This should result in a block number which does have finalized blocks.

With the new query we are (probably?) reading data from a table which only contains information on blocks which are already finalized. This would mean that instead of adding `1` to the largest block number, we could remove the subtraction of `65`.

In any case, the only thing that needs to be prevented is to include blocks in the block range which contain settlements which do not have all the required information in the database yet. So I would opt for not adding `+ 1` and leaving the `- 65`in place. In the future, we should check my argument above and remove the `- 65` if possible.

Reply:
Ok, makes sense. I also had a look at the code and saw this `REORG_THRESHOLD = 65` correction, so probably the +1 wouldn't really change much here.